entries/a/apnic.net.json

```
@@ -0,0 +1,12 @@
+{
+  "APNIC": {
+    "domain": "apnic.net",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://help.apnic.net/s/article/How-do-I-enable-disable-two-factor-authentication",
+    "keywords": [
+      "utilities"

```

```suggestion
      "other"
```

`other` seems to be a more appropriate category, as ARIN is also listed in `other`